### PR TITLE
Fix release workflow

### PR DESCRIPTION
**Pull request by orzeh**
Should fix problems with version drift between git tag and Bintray. Should also publish docs to `gh-pages` branch.

### CODE REVIEW

File: gradle/release.gradle [hunk 1]
```diff
@@ -11,10 +11,9 @@ buildscript {
     }
 }
 
-ext.repo = Grgit.open(rootDir)
+apply plugin: org.ajoberstar.gradle.git.base.GrgitPlugin
 
 release {
-    grgit = Grgit.open(rootDir)
     versionStrategy Strategies.FINAL
     defaultVersionStrategy = Strategies.FINAL
```
Comment (cdancy) [inline gradle/release.gradle]: I believe previously this was Strategies.SNAPSHOT and I changed it to FINAL? Is that still Ok?

**Comment by cdancy** [comment]
Just deleted the v3.0.10 tag as well as the github-release and renamed it to v3.0.11. Hopefully that doesn't mess anything up.

**Comment by cdancy** [comment]
@orzeh just got an update that `v3.0.12` was just released. Are you playing around trying to fix things?

**Comment by cdancy** [comment]
^^^ @bmuschko maybe that's you playing around? Think we need to get things straightened out and fast otherwise we risk the integrity of this project and lose user confidence.

**Comment by bmuschko** [comment]
@cdancy It's not me, folks. I haven't touched the project for a while. 

@orzeh @cdancy Can you guys please coordinate? Maybe we can set up [Gitter](https://gitter.im/) for better collaboration. That'll make it easier to communicate. Thoughts?

**Comment by bmuschko** [comment]
@orzeh @cdancy I created a Gitter chat room and added a badge to the README file. Let's collaborate there. I am sure it will make things much easier.

File: gradle/release.gradle (hunk 1 above)
Comment (orzeh) [inline gradle/release.gradle]: @cdancy  should be `Strategies.SNAPSHOT`

**Pull request merged**